server/actions/createMember.js

```diff
@@ -0,0 +1,23 @@
+import config from 'src/config'
+import graphQLFetcher from 'src/server/util/graphql'
+
+export default async function createMember(id, inviteCode) {
+  let result
+  try {
+    result =
+      await graphQLFetcher(config.server.echo.baseURL)({
+        query: `
+          mutation($id: ID!, $inviteCode: String!) {
+            createMember(id: $id, inviteCode: $inviteCode) {
+              id
+              chapterId
+            }
+          }
+        `,
+        variables: {id, inviteCode},
+      })
+  } catch (error) {
+    throw error
+  }
+  return result
```

Comment:
Let's make sure that the value resolved is the new member data. So you might want to return `result.data.createMember`.